### ISSUE TITLE
Don't check Forge URLs if the game isn't hosted there

### ISSUE THREAD

**Issue by mkahvi** [League-of-Foundry-Developers/FoundryVTT-Deprecated-Modules#4]
Why is this assuming I am on Forge?
```
forge-vtt.com/api/bazaar/manifest/chat-indicators:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
```

Simple check on `window.location.href` will tell you checking Forge links is pointless, and they don't support cross-site requests with their API to use it for anything else.

**Comment by BradfordC** [comment]
The reason I'm using the Forge is because it's the only source I can use for getting up to date manifests for modules, which I need to see if a module has an 0.8.x update ready. Foundry itself doesn't provide access to those files with an API, and there's too much variability in how manifest URLs are defined in manifests that I can't reliably use them either.

Strange that it's not working for you, though. I tested it on both a Windows and a Linux machine, and it's worked fine for both. Could you answer a few questions to help me figure out what's going on?
* What version of Foundy is this happening on, and what type of computer/server is it installed on?
* Is chat accessibility indicators the only module with the issue, or are all of them doing the same thing?
* In the compatibility checker window, are the icons stuck in the loading animation, or are they showing something else?
* How many active modules did you have when you ran the check?

**Comment by mkahvi** [comment]
I feel a bit dumb now, while going over the extra info you requested, I realized I had XSS script blocker acting up on this. I thought I had disabled it for Foundry but no, that seems to have been the cause for the errors. Sorry for troubling you.

Here's the requested info anyway:

- Foundry 0.7.10, Win10, running via Node 14.17.0. Client via Chrome on same computer.
- About 90% of modules have the issue.
- When I reported the issue, almost all of them were stuck with the spinning icon, now only one has that (the game system), the rest have error auto-checking manifest.
- 62
